packages/types/src/appearance.ts

```diff
@@ -830,4 +831,8 @@ export type Appearance<T = Theme> = T & {
    * Theme overrides that only apply to the `<Checkout />` component
    */
   checkout?: T;
+  /**
+   * Theme overrides that only apply to the `<PaymentSources />` component
+   */
+  paymentSources?: T;
```

Comment:
hm, I wouldn't expect this to have its own appearance theme. Wouldn't this inherit the appearance provider from the component its placed within?